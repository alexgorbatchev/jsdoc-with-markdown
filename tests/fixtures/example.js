/**
This is a description.
Line 1.
Line 2.
Line 3.

## Example

```js
function hello() {
  console.log('Hi!');
}
```

Last line.
*/
