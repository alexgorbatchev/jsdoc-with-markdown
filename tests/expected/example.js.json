[
  {
    "type": "paragraph",
    "value": "This is a description. Line 1. Line 2. Line 3."
  },
  {
    "type": "paragraph",
    "value": "## Example"
  },
  {
    "type": "code",
    "value": "```js\nfunction hello() {\n  console.log('Hi!');\n}\n```"
  },
  {
    "type": "paragraph",
    "value": "Last line."
  }
]